controllers/movement.py

```diff
@@ -239,6 +239,9 @@ def set_input(
         local: drive reletive to robot frame rather than global frame
         override: override driver lockout
         """
+        if is_red():
+            vx = -vx
+            vy = -vy
```

Comment:
does this work correctly for autonomous? is there a reason it's being handled in this controller rather than wherever the inputs are coming from?

Reply:
This feature was added in another pr a few days ago, I have removed it from this one. The autonomous doesn't use set_inputs